### ISSUE TITLE
Cannot map cells to reference for human kidney

### ISSUE THREAD

**Issue by juechenyang** [satijalab/azimuth#84]
Hi

I am using **docker version(0.4.3)** of Azimuth to run my data for **human kidney reference**. However, when I tried to **map cells to reference**, some errors always jump out:

![image](https://user-images.githubusercontent.com/18729321/140187292-6a3e4534-ae47-4bc0-a51c-4845ac57561d.png)

I can run my data using **online version of Azimuth** without issue. Would you mind help me with this? Thank you!

**Comment by andrewwbutler** [comment]
Hi,

It's not clear exactly how you ran the docker container but my guess is that you need to set the option `Azimuth.map.ndims = 100`. You can check the rest of the configuration settings for that reference present in the online version [here](https://github.com/satijalab/azimuth-references/blob/master/human_kidney/reference/config.json).

**Comment by juechenyang** [comment]
Hi

Thank you for your quick response. By setting `Azimuth.map.ndims = 100`, the issue has been fixed. However I encountered a new error:
![image](https://user-images.githubusercontent.com/18729321/140236244-7e12354c-def5-4eee-8214-b8352a893b26.png)
I wonder if this can also be fixed by **configure other settings** or **there is some problems** in my data

**BTW**, If I want to increase the maximum upload size limit, should I change the setting of  `"shiny.maxRequestSize": 1048576000`?
